src/github.rs

```diff
@@ -13,7 +13,8 @@ struct GithubOauthResponse {
 
 #[derive(Deserialize, Debug)]
 struct GithubUserResponse {
-    pub name: String,
+    pub name: Option<String>,
+    pub id: i64,
```

Comment:
I realize GitHub returns this as an int currently, but best practices for foreign primary keys is to treat them as strings. That ensures that we're insulated from any potential API changes on the other end.

Reply:
That's a good point. Done.